vllm/v1/core/scheduler.py

```diff
@@ -345,6 +350,7 @@ def _make_cached_request_data(
         request: Request,
         new_block_ids: List[int],
         num_computed_tokens: int,
+        *,
```

Comment:
Why is this needed?

Reply:
oh i believe I forgot removing this from previous commit